src/dataflow-types/src/client.rs

```diff
@@ -32,11 +32,16 @@ use repr::{Row, Timestamp};
 #[derive(Clone, Debug, Serialize, Deserialize)]
 pub enum Command {
     /// A compute command.
-    Compute(ComputeCommand),
+    Compute(ComputeCommand, ComputeInstanceId),
```

Comment:
the instance id is the first argument of all methods, should it also be the first argument here? (i don't care, just a thought)

Reply:
Interesting! It breaks some visual symmetry with the Storage commands, but .. it's a good point (the methods got written after this was).